plugins/modules/system/sudoers.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+
+# Copyright: (c) 2019, Jon Ellis (@JonEllis) <ellis.jp@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: sudoers
+short_description: Manage sudoers files
+version_added: "4.3.0"
+description:
+  - This module allows for the manipulation of sudoers files.
+author:
+  - "Jon Ellis (@JonEllis0) <ellis.jp@gmail.com>"
```

Comment:
@JonEllis you probably want to fix this in a follow-up PR:
```suggestion
  - "Jon Ellis (@JonEllis) <ellis.jp@gmail.com>"
```
(This needs to be fixed in `main` first before we can backport it.)